src/util/utils.ts

```diff
@@ -2,3 +2,71 @@ import { format } from 'date-fns'
 
 export const toLocalTime = (date: Date): String =>
   format(date, 'MMM d, yyyy h:mm aa')
+
+export async function snooze(ms: number): Promise<void> {
+  return await new Promise(resolve => setTimeout(resolve, ms))
+}
+
+type AsyncFunction = () => Promise<any>
+
+export async function asyncWaterfall(
+  asyncFuncs: AsyncFunction[],
+  timeoutMs: number = 5000
```

Comment:
Needs not be an export as it's the default of the function. Exporting it doesn't make it useful for others to use because it would just override the default with the same value. Consumers of the function are welcome to override it.

Reply:
Yeah, I know users can override it, but the intention of the comment is more to ensure any new additions to this file or elsewhere are unified in using that 5000 as a default, magic numbers, especially repeated ones tend to make me uneasy.

Reply:
Ok. So not so much that it's exported but more that it's documented. I'll quickly add that and merge.